packages/clerk-js/src/core/clerk.ts

```diff
@@ -566,7 +590,9 @@ export class Clerk implements ClerkInterface {
   public mountOrganizationList = (node: HTMLDivElement, props?: OrganizationListProps) => {
     this.assertComponentsReady(this.#componentControls);
     if (disabledOrganizationsFeature(this, this.#environment)) {
-      console.info(warnings.cannotRenderAnyOrganizationComponent('OrganizationList'));
+      if (this.#instanceType === 'development') {
+        throw new Error(warnings.cannotRenderAnyOrganizationComponent('OrganizationList'));
```

Comment:
🙃 We have a "ClerkRuntimeError" class we could take advantage of.

Reply:
🔧

Reply:
`ClerkRuntimeError` also requires a code for localization. This makes no sense in this case as this is a dev only error.